core/common/src/main/scala/net/liftweb/common/HList.scala

```diff
@@ -116,7 +96,7 @@ object HLists {
    * Carries the information about the type of this element, plus the `HList`
    * type of the rest of the list.
    *
-   * You can use `:+:` to make this `HList` longer:
+   * You can use `[[HListMethods.:+: :+:]]` to make this `HList` longer:
    *
    * {{{
```

Comment:
I actually waited until the other docs PR for `lift-common` went in, so these docs should all be up-to-date, I think. Did you see an issue? I kind of had to move the tweaks I'd made for lift-formality into the already-properly-doced file, so it's possible I missed something.